app/services/adviser/teaching_subjects.rb

```diff
@@ -0,0 +1,32 @@
+class Adviser::TeachingSubjects
+  SUBJECTS_IDS_TO_EXCLUDE = [
+    '6b793433-cd1f-e911-a979-000d3a20838a', # Art
+    'a62655a1-2afa-e811-a981-000d3a276620', # Media studies
+    'bc68e0c1-7212-e911-a974-000d3a206976', # No preference
+    'bc2655a1-2afa-e811-a981-000d3a276620', # Other
+    'ae2655a1-2afa-e811-a981-000d3a276620', # Physics with maths
+    'ba2655a1-2afa-e811-a981-000d3a276620', # Vocational health
+  ].freeze
```

Comment:
Since we are getting more of these specific IDs, can you move to yaml file and load then into this constants. And also move the worker ones as well? I prefer to have all those ids organised in one place. If you can move those comments into keys that would be good as well. Then you can just load the file and call the values of the hash.